interface/orders/procedure_provider_edit.php

```diff
@@ -171,7 +171,7 @@ function invalue($name)
                                         <p><?php echo xlt("For detailed instructions close the 'Enter Provider Details' popup and click on the Help icon on the main form. ");?><i class="fa fa-question-circle" aria-hidden="true"></i></p>
                                     </div>
                                 </div>
-                                <div class="col-sm-6">
+                                <div class="col-sm-6 mt-3">
                                     <div class="clearfix">
                                         <div class="label-div">
                                             <label class="control-label" for="form_npi"><?php echo xlt('NPI'); ?>:</label> <a href="#npi_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
```

Comment:
`control-label` is no longer a class in Bootstrap. This should be replaced with `col-form-label`.